src/daemon/dlt_daemon_common.c

```diff
@@ -676,6 +767,60 @@ DltDaemonApplication *dlt_daemon_application_add(DltDaemon *daemon,
 #ifdef DLT_LOG_LEVEL_APP_CONFIG
     application->num_context_log_level_settings = 0;
     application->context_log_level_settings = NULL;
+#endif
+#if DLT_TRACE_LOAD_CTRL_ENABLE
+    if (application->trace_load_settings == NULL) {
+        DltTraceLoadSettings* pre_configured_trace_load_settings = NULL;
+        int num_settings = 0;
+        DltReturnValue rv = dlt_daemon_find_preconfigured_trace_load_settings(
```

Comment:
if it is possible, kindly please using the meaning name instead of "rv"

Reply:
RV is pretty common for return value

Reply:
changed to find_trace_settings_return_value